### ISSUE TITLE
Remove Spigot support in favor of Paper

### ISSUE THREAD

**Issue by Arnuh** [Arnuh/EchoPet#139]
In preparation for hard fork, would rather use Paper over Spigot post hard fork. Also allows me to remove spigot testing.

**Comment by Arnuh** [comment]
Partially done, need to readd maven repo publishing and see if I messed up anything with my gradle conversion

**Comment by johngonole** [comment]
Does this mean that in the future this won't plugin won't be able to run on servers using spigot?

**Comment by Arnuh** [comment]
> Does this mean that in the future this won't plugin won't be able to run on servers using spigot?

Yes, will likely start with the use of adventure api in a rework of the text/language configuration in the plugin. This change itself wouldn't restrict the plugin to Paper but I expect future changes to start using the Paper API.
